doc/ref_model/chapters/chapter09.rst

```diff
@@ -220,6 +220,229 @@ consider that this would be done by a different mechanism for Capacity Managemen
 Configuration and Lifecycle Management section deals with the changes being made to the infrastructure hardware,
 software, and management components (e.g. changing of number of hypervisor hosts from 10 to 12).
 
+
+
+Telemetry and Observability
+---------------------------
+
+Operating complex distributed systems, such as a Telco network, is a demanding and challenging task that is continuously
+being increased as the network complexity and the production excellence requirements grow. There are multiple reasons
```

Comment:
"... production excellence" can mean different things to different folks.
What does "production excellence" look like in the cloud era or Kubernetes? Both recognise failure.

Reply:
@pgoyal01, please check the new wording